user/customizing-the-build.md

```diff
@@ -523,12 +524,27 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+#### Conditionally Allowing Jobs to Fail
+
+Allowed failures can include a [condition](https://docs.travis-ci.com/user/conditional-builds-stages-jobs#conditionally-allowing-jobs-to-fail) using the key `if`.
+
+For example, the following would allow the job using `rvm: 1.9.3` to fail
+only on the `master` branch:
+
+```yaml
+jobs:
+  allow_failures:
+  - rvm: 1.9.3
+    if: branch = master
+```
+{: data-file=".travis.yml"}
+
 #### Matching Jobs with `allow_failures`
 
 When matching jobs against the definitions given in `allow_failures`, _all_
-conditions in `allow_failures` must be met exactly, and
-all the keys in `allow_failures` element must exist in the
-top level of the build matrix (i.e., not in `matrix.include`).
+attributes specified on a row in `allow_failures` must be met exactly, and all
```

Comment:
Here, "on a row" is probably incorrect. If there are more than one attribute for a condition to allow failures, it would span multiple rows:

```yaml
jobs:
  allow_failures:
  - os: linux
    rvm: 2.7.0
  - os: linux
    rvm: 2.6.2
```

Reply:
i think talking about rows is kinda misleading anyway, if we have the much more fitting term "job". i'll change that.